packages/notifi-dapp-example/src/hooks/useNotifiRouter.ts

```diff
@@ -53,12 +53,16 @@ export const useNotifiRouter = () => {
 
   useEffect(() => {
     if (loginError) {
-      handleRoute('/');
       if (selectedWallet) {
         wallets[selectedWallet].disconnect();
       } else if (injectiveSelectedWallet) {
         injectiveWallets[injectiveSelectedWallet].disconnect();
       }
+      setTimeout(() => {
+        /** NOTE: preserve 0.5s to make sure cache is cleaned before route back to root dir*/
+        console.log(`LOGIN ERROR: ${JSON.stringify(loginError)}`);
+        handleRoute('/');
+      }, 5000);
```

Comment:
I noticed that the note mentions a 0.5-second delay to ensure the cache is cleaned before routing back to the root directory. However, the code currently has a 5-second delay. Could you please confirm if this is intentional or if it should be 500 milliseconds instead of 5000 milliseconds as per the note?

cc @eric-notifi

Reply:
Nice catch!! I was gona do 500. Thanks @ankush-notifi.